### PR TITLE
Picard addition

### PR DESCRIPTION
**Pull request by akshaaatt**
Picard finally makes its official way in with the tagger.

### CODE REVIEW

**Comment by amCap1712** [comment]
I tried it locally. Works splendidly! There is room for some improvements though. The Picard's logo does not make it clear what the feature is for. I would like if it could be made clearer. Maybe rearrange the buttons and add Send to Picard text next to icon?

Secondly, I think we should send the original values of various tag fields of to Picard not the our suggested ones.

**Comment by akshaaatt** [comment]
@amCap1712 Thanks for the suggestions. As discussed, we will make the UX better with a final PR. The barcode scanner idea has been modified and the picard feature has been made available in the Release Activity for all the releases :)
The Onboarding idea will be incorporated asap in the separate PR.

**Comment by phw** [comment]
Once this is available I'd also be happy to add a tutorial for how to use this to https://picard-docs.musicbrainz.org/

**Comment by akshaaatt** [comment]
@phw Exciting! Thanks for bringing up this addition. Looking forward to collaborating more in the future :)

File: app/src/main/java/org/metabrainz/mobile/util/Utils.kt [hunk 1]
```diff
@@ -31,6 +45,53 @@ object Utils {
...
+        val url = String.format(
+            App.PICARD_OPENALBUM_URL, ipAddress,
+            UserPreferences.preferencePicardPort, uriEncode(releaseMBID)
+        )
+
+        CoroutineScope(context = Dispatchers.Default).launch {
+            var connection: HttpURLConnection? = null
+            try {
+                val u = URL(url)
+                connection = u.openConnection() as HttpURLConnection?
```
Comment (amCap1712) [inline app/src/main/java/org/metabrainz/mobile/util/Utils.kt:67]: I think its better to do the request using `OkHttp`.

Comment (akshaaatt) [reply app/src/main/java/org/metabrainz/mobile/util/Utils.kt]: I agree using `OkHttp` makes more sense. I have made the relevant changes!

**Review by amCap1712 — APPROVED** [review]
LGTM. Awesome work!

**Pull request merged**